modules/xmpp/JingleSessionPC.js

```diff
@@ -2670,19 +2688,18 @@ export default class JingleSessionPC extends JingleSession {
             );
 
         const containsNewSSRCs = sdpDiffer.toJingle(add);
+        const addedSsrcInfo = getSignaledSourceInfo(sdpDiffer);
 
-        if (containsNewSSRCs) {
-            const sourceInfo = getSignaledSourceInfo(sdpDiffer);
-
+        if (containsNewSSRCs && addedSsrcInfo.ssrcs.length) {
```

Comment:
same question as in the `removedAnySSRCs` case, why would it contain new sources while actually there's 0 SSRCs?

Reply:
Same as above